openff/toolkit/tests/test_molecule.py

```diff
@@ -3603,6 +3604,31 @@ def test_assign_fractional_bond_orders(self):
                     # fbo2 = [bond.fractional_bond_order for bond in molecule.bonds]
                     # np.testing.assert_allclose(fbo1, fbo2, atol=1.e-4)
 
+    @requires_ambertools
+    @requires_openeye
+    @pytest.mark.parametrize("model", ["AM1-Wiberg", "am1-wiberg"])
```

Comment:
(non-blocking) This is a well-written PR, but I wish we had a less time-intensive way to test it. We might consider making this one a `runslow`
```suggestion
    @pytest.mark.slow
    @pytest.mark.parametrize("model", ["AM1-Wiberg", "am1-wiberg"])
```